rwengine/src/engine/GameData.cpp

```diff
@@ -522,14 +522,19 @@ bool GameData::loadModel(ModelID model) {
     return true;
 }
 
-void GameData::loadIFP(const std::string& name) {
+void GameData::loadIFP(const std::string& name, bool cutsceneAnimation) {
     auto f = index.openFile(name);
 
     if (f.data) {
         LoaderIFP loader;
         if (loader.loadFromMemory(f.data.get())) {
-            animations.insert(loader.animations.begin(),
-                              loader.animations.end());
+            if (!cutsceneAnimation) {
```

Comment:
I would swap the if/elses. Makes the code clearer.

```suggestion
            if (cutsceneAnimation) {
                animationsCutscene.insert(loader.animations.begin(),
                                          loader.animations.end())
            } else {
                            animations.insert(loader.animations.begin(),
                                  loader.animations.end());
            }
```